tests/test_classifier/test_classification_report.py

```diff
@@ -178,3 +170,46 @@ def test_isclassifier(self):
 
         with pytest.raises(yb.exceptions.YellowbrickError, match=message):
             ClassificationReport(LassoCV())
+
+    def test_support_count_class_report(self):
```

Comment:
Thank you for adding both of these tests - very nice work! I reviewed the baseline images as well, and they're also solid!